src/main/webapp/scripts/itinerary-overview.js

```diff
@@ -227,6 +225,50 @@ function displayRoute(start, waypoints, end) {
     );
 }
 
+/**
+ * Method that changes the travel mode that the user will be using on their trip.
+ * @param travelMode{String} one of the accepted modes of travel by displayRoute() (DRIVING,WALKING,BICYCLING,TRANSIT)
+ */
+function changeTravelMode(travelMode){
+    currentMode = travelMode;
+    displayRoute(start, waypoints, end);
+}
+
+function changeTravelModeDriving(){
+    changeTravelMode("DRIVING");
```

Comment:
I feel like these functions can be refactored into one function where you pass the mode as an argument, without having to repeat code

Reply:
Didn't know how to do it at first so I just got a working solution because it doesn't let me put "" inside the function. Should be better now though